src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java

```diff
@@ -241,15 +245,88 @@ private void ensureWithinRange(@NotNull Position position) throws TileOutOfRange
             throw new TileOutOfRangeException(position, xRange, yRange);
     }
 
+    private void setRunning(boolean newState){
+        if(this.running && !newState){
+            running = false;
```

Comment:
why not "running = newState"?
When we're in the if clause, newState must be false.

Reply:
IDE, said to simplify